lisa/schema.py

```diff
@@ -207,14 +207,43 @@ class Parent:
 
 @dataclass_json()
 @dataclass
-class Extension:
+class ExtensionV1:
     """
     deprecated. Use paths directly
     """
 
     paths: List[str] = field(default_factory=list, metadata=metadata(required=True))
 
 
+@dataclass_json()
+@dataclass
+class Extension:
+    path: str
+    name: Optional[str] = None
+
+    @classmethod
+    def from_raw(cls, raw_data: Any) -> List["Extension"]:
+        results: List[Extension] = []
+
+        if isinstance(raw_data, Dict):
+            # for compatibility, convert extension to list of strings
+            extension_v1: ExtensionV1 = ExtensionV1.schema().load(  # type: ignore
```

Comment:
What does the comment mean in the line 230 and 241?

Reply:
improved